package.json

```diff
@@ -527,19 +527,28 @@
     "start": "stripes serve",
     "build": "stripes build",
     "lint": "eslint .",
-    "test": "echo 'No unit tests implemented'",
+    "test": "stripes test karma",
     "test-int": "stripes test nightmare"
   },
   "devDependencies": {
+    "@bigtest/interactor": "^0.9.1",
+    "@bigtest/mirage": "^0.0.1",
+    "@bigtest/mocha": "^0.5.2",
+    "@bigtest/react": "^0.1.2",
     "@folio/eslint-config-stripes": "^3.2.1",
     "@folio/stripes": "^1.1.0",
-    "@folio/stripes-cli": "^1.5.0",
+    "@folio/stripes-cli": "^1.6.0",
+    "@folio/stripes-core": "^2.16.1000442",
```

Comment:
I made the same mistake, you should use stripes core like `^2.16.1` or something, from `npm-folio` registry.
https://github.com/folio-org/ui-orders/pull/35#discussion_r227049002

Reply:
Done